test/unit/pdf_find_controller_spec.js

```diff
@@ -156,6 +156,36 @@ function testSearch({
   });
 }
 
+function testEmptySearch({ eventBus, pdfFindController, state }) {
+  return new Promise(function (resolve) {
+    const eventState = Object.assign(
+      Object.create(null),
+      {
+        source: this,
+        type: "",
+        query: null,
+        caseSensitive: false,
+        entireWord: false,
+        phraseSearch: true,
+        findPrevious: false,
+        matchDiacritics: false,
+      },
+      state
+    );
+    eventBus.dispatch("find", eventState);
+
+    eventBus.on(
+      "updatefindcontrolstate",
+      function onUpdatefindcontrolstate(evt) {
+        eventBus.off("updatefindcontrolstate", onUpdatefindcontrolstate);
```

Comment:
This function isn't actually testing the "correct" thing, since it doesn't wait for searching to stop before checking the expected value. Hence I'd expect the following instead:
```suggestion
      function onUpdateFindControlState(evt) {
        if (evt.state !== FindState.NOT_FOUND) {
          return;
        }
        eventBus.off("updatefindcontrolstate", onUpdateFindControlState);
```